### PR TITLE
Fix/remove magic strings

### PR DESCRIPTION
**Pull request by ForeverYoung1208**

### CODE REVIEW

File: src/components/AppHeader/AppHeader.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Toolbar, useColorModeContext } from '@eten-lab/ui-kit';
 
 import { useAppContext } from '@/hooks/useAppContext';
 import { RouteConst } from '@/constants/route.constant';
+import { ColorThemes } from '../../constants/common.constant';
```
Comment (lucky-gru) [inline src/components/AppHeader/AppHeader.tsx]: `import { ColorThemes } from '../../constants/common.constant';`

to 

`import { ColorThemes } from '@/constants/common.constant';`

File: src/components/FeedbackInput/FeedbackInput.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { useHistory } from 'react-router-dom';
 
 import { useAppContext } from '@/hooks/useAppContext';
 import { CrowdBibleUI, MuiMaterial } from '@eten-lab/ui-kit';
+import { FeedbackTypes } from '../../constants/common.constant';
```
Comment (lucky-gru) [inline src/components/FeedbackInput/FeedbackInput.tsx]: plz use absolute path

File: src/components/MapTranslatorTabs/MapTabContent.tsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ import {
 import { useAppContext } from '../../hooks/useAppContext';
 import { useMapTranslationTools } from '../../hooks/useMapTranslationTools';
 import { langInfo2String } from '../../utils/langUtils';
+import { FeedbackTypes } from '../../constants/common.constant';
 const { Box, styled, CircularProgress } = MuiMaterial;
```
Comment (lucky-gru) [inline src/components/MapTranslatorTabs/MapTabContent.tsx:19]: use absolute path

File: src/hooks/useDefinition.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { useCallback } from 'react';
 import { useAppContext } from '@/hooks/useAppContext';
 
 import { LanguageInfo } from '@eten-lab/ui-kit';
+import { FeedbackTypes } from '../constants/common.constant';
```
Comment (lucky-gru) [inline src/hooks/useDefinition.ts]: plz use absolute path

File: src/components/MapTranslatorTabs/MapTabContent.tsx (hunk 3 above)
Comment (ForeverYoung1208) [reply src/components/MapTranslatorTabs/MapTabContent.tsx:19]: > use absolute path

done.

File: src/pages/DataTools/GraphViewer/SearchNodePage.tsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ import { Like } from 'typeorm';
 import { useGlobal } from '@/src/hooks/useGlobal';
 import { initialState, reducer } from '@/src/reducers';
 import { useHistory } from 'react-router';
-import { useAppContext } from '../../../hooks/useAppContext';
+import { useAppContext } from '@/src/hooks/useAppContext';
```
Comment (lucky-gru) [inline src/pages/DataTools/GraphViewer/SearchNodePage.tsx:8]: @/src/hooks => @/hooks
@/src/utils => @/utils
...

File: src/components/MapTranslatorTabs/MapTabContent.tsx (hunk 3 above)
Comment (ForeverYoung1208) [reply src/components/MapTranslatorTabs/MapTabContent.tsx:19]: done. Changed all paths to relative.

**Review by lucky-gru — APPROVED** [review]

**Pull request merged**